### ISSUE TITLE
Change name 

### ISSUE THREAD

**Issue by tanaes** [jdereus/labman#175]
'Labman' is disconcertingly male-directed (I've heard 'labperson' a few times so I know I'm not the only one). 

'[Manager](https://www.etymonline.com/word/manage)' is derived from the latin 'manus' for hand, and ultimately pertains to the 'by-hand' nature of work. 

'[Man](https://www.etymonline.com/word/man)' is germanic in origin, related more directly to maleness. 

Perhaps it's not too late to propose an alternative?

**Comment by josenavas** [comment]
Not at all - and now that you pointed that out I definitively want to change it.

Can we get some alternatives here?
My 2 cents: LabHelper

**Comment by tanaes** [comment]
Other suggestions in the same vein: 

Lab[Menagerie](https://www.etymonline.com/word/menagerie)

Lab[Mnemon](https://www.etymonline.com/word/mnemonic)

**Comment by josenavas** [comment]
Given that the publication is going to be centered around the EMP protocols, EMPLab?

**Comment by josenavas** [comment]
Also, the easiest one, the origin of actual LabMan:

LabManager

**Comment by antgonza** [comment]
Following the tradition of ii -> Labiing?

**Comment by c1carpenter** [comment]
Austin suggested TIIME? 
‘tracking information in microbiome experiments

**Comment by adswafford** [comment]
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMan)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMan/vote)
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabManager)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabManager/vote)
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMenagerie)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMenagerie/vote)
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMnemon)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/LabMnemon/vote)
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/EMPLab)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/EMPLab/vote)
[![](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/TIIME)](https://api.gh-polls.com/poll/01C65T1JB6DNCM8FG166EC2ERV/TIIME/vote)

**Comment by adswafford** [comment]
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMan)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMan/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMenagerie)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMenagerie/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMnemon)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabMnemon/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabManager)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/LabManager/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/EMPLab)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/EMPLab/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/Labiing)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/Labiing/vote)
[![](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/TIIME)](https://api.gh-polls.com/poll/01C65T4XGMYF3CN5WPRPT3AA38/TIIME/vote)

**Comment by adswafford** [comment]
Found this for polls btw: https://app.gh-polls.com

**Comment by adswafford** [comment]
FYI-
LabManager:
http://www.labmanager.com
http://labsysgrp.com/lab-manager-v7/

LabMan:
https://www.labmanautomation.com
https://www.wright.edu/event/labman/labman-2017
https://www.ncbi.nlm.nih.gov/pubmed/8013891

**Comment by antgonza** [comment]
Changing to high as it will be great to have a final name for final deploy; what do you think @AmandaBirmingham ? 

Some new options (remember q is pronounced as CH):
- qaqalaca
- qalupa
- qamp -- lab-qamp!

**Comment by wasade** [comment]
* qamnit
* qeese
* qirp-qirp
* enqiladas

**Comment by adswafford** [comment]
How about
- Sample Handling Information and Tracking System
- Processing Operation Organization Program
- Workflow Tracking Framework

or rather acronyms of these?

**Comment by tanaes** [comment]
ooooh I *really* like "lab-qamp"!

**Comment by antgonza** [comment]
I can work on changing the schema and other things once we get a final name. @AmandaBirmingham, could you ask the wetlab if they have a preference and/or care?

**Comment by mortonjt** [comment]
We have officially decided to rename this to lab-qamp

🎆 

You have 48 hours to object to this decision in this thread

**Comment by charles-cowart** [comment]
I'll ask the wet-lab members for input. If someone can assign me to this, I'd appreciate it.

**Comment by antgonza** [comment]
@jdereus, I can't assign to @charles-cowart because he's not a collaborator in the team and I can't add cause I don't have permissions; could you?

**Comment by charles-cowart** [comment]
Wet-lab users asked to suggest possible new names using an online survey:

https://charlescowart.typeform.com/to/dfvF6n

**Comment by charles-cowart** [comment]
Final results for Naming poll included here.
[Naming Poll for Labman.pdf](https://github.com/jdereus/labman/files/2577778/Naming.Poll.for.Labman.pdf)

**Comment by AmandaBirmingham** [comment]
It is probably a good idea to check with Rob just to ensure there are none he would veto; I can do that at lunch today.

**Comment by charles-cowart** [comment]
Thanks Amanda! Gail mentioned earlier that Greg has the final say; is this
okay with you and Rob as well?

**Comment by charles-cowart** [comment]
The following suggestions were also made in the poll, but not voted on:

- DeepQiit
- Qiitmaker
- Qopper
- Qiimney
- Hive Mind

**Comment by AmandaBirmingham** [comment]
Ok with me. Can't speak for Rob on that but can ask.

**Comment by AmandaBirmingham** [comment]
Just to confirm: in this poll, the closer the ranking is to 10, the more people liked the name, right?

**Comment by charles-cowart** [comment]
Yes, the higher average is to 10, the more people liked it, or the more
strongly they liked it.

**Comment by AmandaBirmingham** [comment]
Per Rob 11/13/2018: He wouldn't veto any of the names voted on in the poll, and he is fine with Greg making the final choice.  He also said that lab-qamp (which wasn't in the poll but was the tech team's preference for a name back in september) was pretty good.

**Comment by AmandaBirmingham** [comment]
I asked Rob: "does the new name chose for labman need to be one NOT already in use for a public software project? This is the requirement we usually would impose for software that would be published, but seems unlikely that labman falls into that class"

He replied: "Ideally, yes ..." and noted "Looks like labmaster is used for a lot of different things (but not commercial LIMS) and would be ok to use" but then noted that it could be considered gendered as well.  

He said his guidance is: **Pick a name not publicly in use for a LIMS and not too embarrassing to put on a public slide.**  When asked if he had a preference between the available names labmaster, labmistress, labgurvi, and lab-qamp, he said he didn't love any of them but they all meet his guidance. He then suggested a couple of other options:

"Looks like labcommand or labcontrol are plausible" [not in use publicly].  "Another direction could be something like PerfectPlates, PerfectPools, etc. .... Looking up synonyms for control, I kind of like labdespot or labtyrant ..."
